doc/manual/dev-06-wal.md

```diff
@@ -60,12 +63,52 @@ This format makes it easy to visually distinguish different parts of the WAL fil
 
 #### Example
 
-To view WAL file details in JSON format:
+1. To view WAL file details in JSON format:
 
 ```bash
 pgmoneta-walinfo -F json /path/to/walfile
 ```
 
+2. To view WAL file details with OIDs in the records translated to object names:
+
+Currently, `pgmoneta-walinfo` supports translating OIDs in two ways,
+1. If the user provided `pgmoneta_user.conf` file, the tool will use it to get the needed credentials to connect to the database cluster and fetch the object names. directly from it.
+```bash
+pgmoneta-walinfo -t -u /path/to/pgmoneta_user.conf /path/to/walfile
+```
+
+2. If the user provided a mapping file that contains the OIDs and the corresponding object names, the tool will use it to translate the OIDs to the object names. This option helps if the user doesn't have the `pgmoneta_user.conf` file or doesn't want to use it.
+```bash
+pgmoneta-walinfo -t -m /path/to/mapping.json /path/to/walfile
+```
+
+User can get the needed info to create the file using these queries:
+```sql
+SELECT spcname, oid FROM pg_tablespace
+SELECT datname, oid FROM pg_database
+SELECT nspname || '.' || relname, c.oid FROM pg_class c JOIN pg_namespace n ON c.relnamespace = n.oid
+```
+
+In either ways, the user should use the `-t` flag to enable the translation. If user provided `pgmoneta_user.conf` file or the mapping file, the tool will do nothing if the `-t` flag is not provided. 
+
+User can create the `pgmoneta_user.conf` file by following the instructions in the [DEVELOPER.md]('../DEVELOPERS.md').
+
+After using this translation feature, the output will change XLOG records from something like this
+`
+Heap2 | 1/D8FFD1C0 | 1/D8FFEB50 | 59 | 59 | 958 | cutoff xid 0 flags 0x03 blkref #0: rel tablespace=1663/dbname=16399/relation=16733 forknum 2 blk 0 blkref #1: rel tablespace=1663/dbname=16399/relation=16733 forknum 0 blk 27597
+`
+
+to this
+`
+Heap2 | 1/D8FFD1C0 | 1/D8FFEB50 | 59 | 59 | 958 | cutoff xid 0 flags 0x03 blkref #0: rel tablespace=pg_default/dbname=mydb/relation=test_tbl forknum 2 blk 0 blkref #1: rel tablespace=pg_default/dbname=mydb/relation=16733 forknum 0 blk 27597
```

Comment:
We don't need the prefix so `pg_default/mydb/test_tbl`.

` rel tablespace=pg_default/dbname=mydb/relation=16733` the same